### PR TITLE
Add: use PyPTO paged attention in Qwen3 decode

### PR DESCRIPTION
**Pull request by Little-oil**
- Add native PyPTO Qwen3-14B paged attention with fused Q/K norm,
  RoPE, V scaling, current-token cache append, and ragged GQA
- Route production `decode_fwd.py` through the PyPTO implementation
  and reuse attention scratch across layers and public-batch windows
- Keep the original handwritten CCE backend, kernel sources,
  regeneration tool, and validation path as standalone references
- Run 24 mixed-core groups with grid-stride task distribution and
  split each five-row GQA group across the two AIV lanes
- Preserve `PRE_LAUNCH=2` at 512-token stack granularity with a
  three-slot transfer ring between QK, softmax, and PV
- Prefetch valid V pages before `SOFTMAX_READY`, ping-pong P through
  two L1 slots, and accumulate P@V in one FP32 tile
- Keep one-, two-, and three-page tails branch-local so absent pages
  issue no cache/probability loads, matmuls, or extra handoffs
- Add dynamic page-layout, Torch golden, cache-canary, codegen, and
  optional raw-timing coverage
- Split the Qwen3-14B model page into `docs/models/qwen3_14b/`, add a
  standalone Page Attention tutorial page, and update the mkdocs nav
  plus the links that pointed at the old path

### CODE REVIEW

File: models/qwen3_14b/test_paged_attention_pypto.py [hunk 1]
```diff
@@ -0,0 +1,1146 @@
...
+    if case.pattern == "zero-query":
+        fixture["q_proj"].zero_()
+        return
+    if case.pattern == "kv-head-id":
+        key[active_pages].zero_()
+        for kv_head in range(NUM_KV_HEADS):
+            value[active_pages, :, kv_head].fill_(float(kv_head + 1))
+            col = slice(kv_head * HEAD_DIM, (kv_head + 1) * HEAD_DIM)
+            fixture["v_proj"][:, col] = float(kv_head + 1) / fixture["inv_rms_states"]
+        return
```
[the remark this answers is not in the document]
Comment (Little-oil) [reply models/qwen3_14b/test_paged_attention_pypto.py:333]: Fixed in `803d69b` — the kv-head-id fixture now uses indexed assignment for both K and V, so it initializes the actual cache tensors instead of PyTorch advanced-index copies.
